spyder/plugins/editor/widgets/editor.py

```diff
@@ -1568,6 +1571,20 @@ def close_all_but_this(self):
         for i in range(0, self.get_stack_count()-1  ):
             self.close_file(0)
 
+    """ Sort open tabs alphabetically """
+    def sort_file_tabs(self):
+        while self.sorted() == False:
+            for i in range(0,self.tabs.tabBar().count()):
+                if( self.tabs.tabBar().tabText(i) > self.tabs.tabBar().tabText(i+1) ):
+                    self.tabs.tabBar().moveTab(i, i+1)
+
+    """ Utility function for sort_file_tabs """

```

Comment:
Please also do the same for this docstring.

Reply:
Made change